### PR TITLE
Clean Up JointState

### PR DESCRIPTION
**Pull request by howard-stearns**
JointState has our own copy of any ivars we want from FBXJoint, instead of keeping a reference to the fbx.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 42cc34cd64dae1e8d9edd129ae0d83f88c420e44 available at: https://gist.github.com/ca8128409bcbb8516106<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5524.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5524.dmg

File: tests/animation/src/RigTests.cpp [hunk 1]
```diff
@@ -79,7 +79,7 @@ void RigTests::initTestCase() {
 
 static void reportJoint(int index, JointState joint) { // Handy for debugging
     std::cout << "\n";
-    std::cout << index << " " << joint.getFBXJoint().name.toUtf8().data() << "\n";
+    std::cout << index << " " << joint.getName.toUtf8().data() << "\n";
```
Comment (sethalves) [inline tests/animation/src/RigTests.cpp]: getName() ?

Comment (howard-stearns) [inline tests/animation/src/RigTests.cpp]: Good catch.

**Comment by hifi-gustavo** [comment]
Build console output for commit: a840a17106475fa284870f6d1c7e7cdcea07316a available at: https://gist.github.com/ca8128409bcbb8516106<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5524.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5524.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 145b730f80d4f52eba5861b53d37d0dcf825f463 available at: https://gist.github.com/ca8128409bcbb8516106<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5524.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5524.dmg

**Pull request merged**